src/index.html

```diff
@@ -7,6 +7,326 @@
+                  </div>
+                </div>
+                <div class="card__stars__sign">Отзывов: 5</div>
+              </div>
+              <div class="card__price">
+                <div class="card__price__sign">Цена:</div>
+                <div class="card__price__price">69 999 грн</div>
+              </div>
+              <a
+              class="card__buy"
+              href="#"
+              
+              >
+              купить
+              </a>
+            </div>
+          </div>
+          <div class="card" >
+            <div class="card__content">
+              <div class="card__image"></div>
+              <div class="card__info">
+                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+              </div>
+              <div class="card__cod">
+                Код товара: 195434
+              </div>
+              <div class="card__stars">
+                <div class="container">
+                  <div class="stars stars--4">
+                    <div class="stars__star"></div>
+                    <div class="stars__star"></div>
+                    <div class="stars__star"></div>
+                    <div class="stars__star"></div>
+                    <div class="stars__star"></div>
+                  </div>
+                </div>
+                <div class="card__stars__sign">Отзывов: 5</div>
+              </div>
+              <div class="card__price">
+                <div class="card__price__sign">Цена:</div>
+                <div class="card__price__price">69 999 грн</div>
+              </div>
+              <a
+              class="card__buy"
+              href="#"
+              
+              >
+              купить
+              </a>
+            </div>
+          </div>
+          <div class="card" >
+            <div class="card__content">
+              <div class="card__image"></div>
+              <div class="card__info">
+                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+              </div>
+              <div class="card__cod">
+                Код товара: 195434
+              </div>
+              <div class="card__stars">
+                <div class="container">
+                  <div class="stars stars--4">
+                    <div class="stars__star"></div>
+                    <div class="stars__star"></div>
+                    <div class="stars__star"></div>
+                    <div class="stars__star"></div>
+                    <div class="stars__star"></div>
+                  </div>
+                </div>
+                <div class="card__stars__sign">Отзывов: 5</div>
+              </div>
+              <div class="card__price">
+                <div class="card__price__sign">Цена:</div>
+                <div class="card__price__price">69 999 грн</div>
+              </div>
+              <a
+              class="card__buy"
+              href="#"
+              
+              >
+              купить
+              </a>
```

Comment:
```suggestion
              <a
                class="card__buy"
                href="#"
              >
                купить
              </a>
```
Что с форматированием html?